src/munge.ts

```diff
@@ -40,6 +41,15 @@ export function disableRemb(sdp: Sdp) {
   disableRtcpFbValue(sdp, 'goog-remb');
```

Comment:
Should we update this to take an `sdpOrAv` while we're at it?

Reply:
udpated. also add UT for disableRemb.